pages/posts/page/[page_number].tsx

```diff
@@ -0,0 +1,92 @@
+import fs from 'node:fs';
+import type { GetStaticPaths, GetStaticProps } from 'next';
+import type { DualPost } from '../../../lib/types';
+import PostPreview from '../../../components/PostPreview';
+import { getAllPosts } from '../../../lib/getAllPosts';
+import { podcastXml } from '../../../lib/podcast';
+import PageWrapper from '../../../components/PageWrapper';
+import Paginator from '../../../components/Paginator';
+import { paginate } from '../../../lib/helpers';
+
+export const getStaticProps: GetStaticProps = async (context) => {
+  const allPosts = await getAllPosts();
+  const whichPage = Number(context.params?.page_number);
+  const thisPagePosts = paginate(
+    allPosts.sort(
+      (a, b) => new Date(b.createdAt).getTime() - new Date(a.createdAt).getTime(),
```

Comment:
i'm pretty sure you can get rid of the sorting here (and in the spanish version of the same page), the `getAllPosts` function sorts it in newest-first order already. might be worth quick double-checking that the order comes out right, but i'm pretty sure it will, and that will tighten up these two files a little bit.